### ISSUE TITLE
run with error "Nan in summary histogram for: transitions_0/grad/hist"

### ISSUE THREAD

**Issue by yulirain** [LopezGG/NN_NER_tensorFlow#6]
Hi, @LopezGG , I run your original code by using the sample data, however, it always has an error "InvalidArgumentError (see above for traceback): Nan in summary histogram for: transitions_0/grad/hist
	 [[Node: transitions_0/grad/hist = HistogramSummary[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"](transitions_0/grad/hist/tag, gradients/AddN_4)]]"

Can you help me ? I am looking forward to your reply. Thank you very much!

**Comment by LopezGG** [comment]
Play with learning rate. This varies based on data

**Comment by SeekPoint** [comment]
so what 's the right setting with the Sample_data?
I got the same error, tried several round, not solve yet

**Comment by LopezGG** [comment]
The setting I have worked for the input mentioned in the paper. It has worked for others as well. Check your tensorflow version as well , maybe ??

**Comment by PARULSHARM** [comment]
I am getting the same error. Played with the learning rate for long time but could not solve the issue. Can you please tell me optimum learning rate ?

Thanks.

**Comment by SeekPoint** [comment]
what 's the solution for sample_data?
